src/AdminRoom.ts

```diff
@@ -126,16 +130,30 @@ export class AdminRoom extends AdminRoomCommandHandler {
         });
     }
 
-    @botCommand("help", "This help text")
+    @botCommand("help", { help: "This help text", category: "general" })
     public async helpCommand() {
         const enabledCategories = [
+            "general",
             this.config.github ? "github" : "",
             this.config.gitlab ? "gitlab" : "",
             this.config.jira ? "jira" : "",
         ];
         return this.botIntent.sendEvent(this.roomId, AdminRoom.helpMessage(undefined, enabledCategories));
     }
 
+    @botCommand("disconnect", { help: "Remove a connection", requiredArgs: ['roomId', 'id'], category: "general" })
```

Comment:
How are we securing these endpoints against not-admins? The admin room is slightly badly named in that it's your personal administration room, not always the bridge admin. We have a bridge admin role in the config that we could use to check with though.

Reply:
> The admin room is slightly badly named in that it's your personal administration room

Oops. Seems like I got bitten by this :)

Addressed in 8842b893751da8eff5a38a045f71605523887bab which requires `connectionManager` permission level (which conveniently exists), also giving these their own config section.